### ISSUE TITLE
Notification importance option

### ISSUE THREAD

**Issue by CodeSpoof** [Futsch1/medTimer#150]
I'd love an option to change the notification importance to high so it gets displayed on top.

**Comment by Futsch1** [comment]
@CodeSpoof See the details about how it was implemented here https://github.com/Futsch1/medTimer/discussions/161

**Comment by CodeSpoof** [comment]
Doesn't seem to work yet. Priority is Standard for both Channels with no option to change to Urgent

**Comment by Futsch1** [comment]
You are using 1.8.9?

**Comment by CodeSpoof** [comment]
Yes, the option to choose the notification channel exists aswell, but the channel has priority capped at standard.

![Screenshot_20240616-220606_Einstellungen~2.png](https://github.com/Futsch1/medTimer/assets/42744253/c81e0380-3f12-43f6-a48f-24460f465b14)

**Comment by Futsch1** [comment]
The change in this notification priority is the pop-up on the screen instead of just in the notification bar. That is what the priority high does.

What else would it be you'd expect?
